src/errors.js

```diff
@@ -82,9 +82,15 @@ export function onlyErrorsWithCode(
   return (error) => {
     let throwError = true;
 
-    if (Array.isArray(codeWanted) && codeWanted.indexOf(error.code) !== -1) {
-      throwError = false;
-    } else if (error.code === codeWanted) {
+    if (Array.isArray(codeWanted)) {
+      if (codeWanted.indexOf(error.code) !== -1) {
+        throwError = false;
+      }
+      if (codeWanted.indexOf(error.errno) !== -1) {
```

Comment:
I added the || code to reduce the repetitiveness of if statements but there was this issue of not having more than 80 columns in a single line. 
It could be done this way, didn't think of that earlier .  
Done.